research/deep_speech/data/dataset.py

```diff
@@ -126,19 +122,20 @@ def _preprocess_data(file_path):
     file_path: a string specifying the csv file path for a dataset.
 
   Returns:
-    A list of tuples (audio_file_path, file_size, transcript) sorted by file
-    size.
+    A list of tuples (wav_filename, wav_filesize, transcript) sorted by
+    file_size.
   """
   tf.logging.info("Loading data set {}".format(file_path))
   with tf.gfile.Open(file_path, "r") as f:
     lines = f.read().splitlines()
-  # Skip the csv header in lines[0].
+  # Skip the tsv header in lines[0].
```

Comment:
what is tsv?

Reply:
tab-separated values? https://en.wikipedia.org/wiki/Tab-separated_values

Reply:
I think csv is more general and understandable. We can use csv here to keep consistent with line 122.